platform/podjs/webpack.config.js

```diff
@@ -9,7 +9,7 @@ export default {
     output: {
```

Comment:
but 200 is fine isn't it? Locally it works, just cypress is acting up I think

Reply:
200 should be fine, but there's something wrong which I can't really place. I made [this question](https://stackoverflow.com/questions/73018360/problems-with-cypress-webpack-wasm-webassembly-instantiate-expected-ma) in SO, no answer so far.

Reply:
I'm not really sure it's exactly cypress. cypress uses webpack as a web server too, it might be related to the MIME type it's serving. But I really don't know, kinda lost here. Maybe we need to create the tests in some other way, or configure webpack-in-cypress in some other way.

Reply:
Yeah I feel you, thanks for the help. Also spent some time trying to figure out whats going on leading nowhere

Reply:
We can meet tomorrow if you want to see if we figure this out. Should not be impossible...

Reply:
Yes I would be very glad!